root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Resources/PublicExceptionMessageResource.resx

```diff
@@ -180,6 +180,9 @@
   <data name="NOT_IMPLEMENTED" xml:space="preserve">
     <value>This processing is not Implemented.</value>
   </data>
+  <data name="NOT_SUPPORTED" xml:space="preserve">
+    <value>This process is not supported .</value>
```

Comment:
@daisukenishino 

We have removed space between "supported" and "." in [PublicExceptionMessageResource.resx  file](https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/162/commits/ce92daac6038eee21bdf31910500b6014ff3d92e#diff-04a2897f285535a9c7049234ba03c52bR184) in [#162](https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/162/commits/ce92daac6038eee21bdf31910500b6014ff3d92e#diff-04a2897f285535a9c7049234ba03c52bR184)

Please review [#162](https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/162) and merge.